doc/api/cli.md

```diff
@@ -751,6 +751,10 @@ Enables report to be generated on uncaught exceptions. Useful when inspecting
 the JavaScript stack in conjunction with native stack and other runtime
 environment data.
 
+### `--set-output-dir=directory`
```

Comment:
My apologies for the bikeshedding, but a name like `--diagnostic-dir=` or `--diagnostic-root=` may be better... basically, something that makes it clear that it's for diagnostic output. Also, the `set` doesn't really match the style of other command-line options.

Reply:
Also, what should be the behavior if the path specified does not exist?

Reply:
Yep I just picked a placeholder name as I was sure there was going to be a better name than I could come up with.

Reply:
> Yep I just picked a placeholder name as I was sure there was going to be a better name than I could come up with.

I once named a cat "Cat" so I'll admit that I'm not really the best person to suggest a better name ;-) 

Thanks for starting this work, btw, it's definitely helpful.